mlflow/store/tracking/abstract_store.py

```diff
@@ -649,19 +651,40 @@ def shut_down_async_logging(self):
             self._async_logging_queue.shut_down_async_logging()
 
     @abstractmethod
-    def log_inputs(self, run_id: str, datasets: Optional[List[DatasetInput]] = None):
+    def log_inputs(
+        self,
+        run_id: str,
+        datasets: Optional[List[DatasetInput]] = None,
+        models: Optional[List[ModelInput]] = None,
+    ):
         """
         Log inputs, such as datasets, to the specified run.
 
         Args:
             run_id: String id for the run
             datasets: List of :py:class:`mlflow.entities.DatasetInput` instances to log
                 as inputs to the run.
+            models: List of :py:class:`mlflow.entities.ModelInput` instances to log
+                as inputs to the run.
 
         Returns:
             None.
         """
 
+    def log_outputs(self, run_id, outputs: Optional[List[ModelOutput]] = None):
```

Comment:
@harupy The name of the arg with type `Optional[List[ModelOutput]]` should be `models`, not `inputs` / `outputs`